config/project/project.yaml

```diff
@@ -0,0 +1,76 @@
+dateModified: 1716932096
+email:
+  fromEmail: $SYSTEM_EMAIL_FROM
+  fromName: 'Viget Craft Starter'
+  replyToEmail: null
+  template: null
+  transportSettings:
+    host: $SYSTEM_EMAIL_HOSTNAME
+    password: $SYSTEM_EMAIL_PASSWORD
+    port: $SYSTEM_EMAIL_PORT
+    useAuthentication: '1'
+    username: $SYSTEM_EMAIL_USERNAME
+  transportType: craft\mail\transportadapters\Smtp
```

Comment:
This assumes we'll use some kind of SMTP service on our projects. 

I've found it to be necessary to get password reset emails to work, even for small clients.

It's super affordable: https://aws.amazon.com/ses/pricing/